presto-product-tests/src/main/java/com/facebook/presto/tests/utils/JdbcDriverUtils.java

```diff
@@ -78,7 +78,11 @@ else if (usingTeradataJdbcDriver(connection)) {
 
     public static boolean usingPrestoJdbcDriver(Connection connection)
     {
-        return getClassNameForJdbcDriver(connection).equals("com.facebook.presto.jdbc.PrestoConnection");
+        String canonicalClassName = getClassNameForJdbcDriver(connection);
```

Comment:
@ebd2 I used the class name because I did not think isinstance would work for a dynamically loaded class.